tests/topotests/bgp_vpnv4_allocnh/test_bgp_vpnv4_allocnh.py

```diff
@@ -139,59 +139,66 @@ def teardown_module(_mod):
     tgen.stop_topology()
 
 
-def bgp_vpnv4_table_check(router, labelList=None):
-    group1 = ["172.31.0.11/32", "172.31.0.20/32", "172.31.0.111/32"]
-    group2 = ["172.31.0.12/32", "172.31.0.15/32"]
-    group3 = ["172.31.0.13/32"]
-    group4 = ["172.31.0.14/32"]
-    group5 = ["192.168.255.0/24", "192.0.2.0/24"]
-    total_group = [group1, group2, group3, group4, group5]
-    for grp in total_group:
-        storedLabelInited = False
-        for entry in grp:
-            dump = router.vtysh_cmd(
-                "show bgp ipv4 vpn {} json".format(entry), isjson=True
-            )
-            for rd in dump:
-                assert (
-                    "prefix" in dump[rd].keys()
-                ), "{0}, {1}, route distinguisher not present".format(
-                    router.name, entry
-                )
-                assert dump[rd]["prefix"] == str(
-                    entry
-                ), "{0}, {1}, ipv4 vpn entry not present".format(router.name, entry)
```

Comment:
please do not remove what you added a few commit ago!

Reply:
done.